setup.py

```diff
@@ -18,7 +19,7 @@
     ],
     entry_points={
         "console_scripts": [
-            "pyfxa-versions = ???:main",
+            "pyfxa-versions = pyfxa_versions:main",
```

Comment:
`-` is not a valid Python identifier character, so I renamed the directory with the code. Thus the name of this project and package is `pyfxa-versions` and the name of the Python module is `pyfxa_versions`.

Reply:
ah, excellent!
tbh I didn't really understand the differentiation between the module and the package naming wrt the directory `__init__.py` drops into. I get it now; `__init__.py` goes into the *module* directory.